Tests/ApolloCacheDependentTests/StarWarsServerTests.swift

```diff
@@ -3,9 +3,64 @@ import XCTest
 import ApolloTestSupport
 import StarWarsAPI
 
+
+protocol TestConfig {
+  func network() -> HTTPNetworkTransport
+}
+
+class DefaultConfig: TestConfig {
+  func network() -> HTTPNetworkTransport {
+    return HTTPNetworkTransport(url: URL(string: "http://localhost:8080/graphql")!,
+                                enableAutoPersistedQueries: false,
+                                useHttpGetMethodForPersistedQueries: false)
```

Comment:
since these both default to false, you can omit the params here. That also helps ensure that nobody changes the default values out from under us.

Reply:
updated

Reply:
oh sorry, I missed that part, updated! thanks!